raspberry/DiscordBot/README.md

```diff
@@ -57,10 +63,27 @@ Then, just run the script
 python main.py
 ```
 
+
+### How to run the client on Start_up
+
+After the virtual environment and dependencies have been installed as described above, a script can be used to atuomatically run the client on start_up.
+
+To do this, the `run_DiscordBot_on_start_up.sh` can be added to the users crontab by:
+
+```
+crontab -e
+```
+Then add the following:
+
+```
+@reboot sleep 30 && ~/bzzz/raspberry/DiscordBot/run_DiscordBot_on_start_up.sh
```

Comment:
Isn't `sleep 30` a bit too much? Have a look at [this](https://raspberrypi.stackexchange.com/questions/78991/running-a-script-after-an-internet-connection-is-established).

Reply:
I tried the link you sent but we get errors in the `main.py` file as it can't find `os.getlogin`.
also prefer the crontab method as it's a more general solution as we don't have to specify specific paths as they will change from one device to anther e.g. /home/bzzz1 or /home/bzzz2